docs/mobile/overview.mdx

```diff
@@ -15,75 +16,60 @@ Geocortex Mobile was designed to be highly configurable out of the box. Geocorte
 
 <DesignerCallout />
 
-## Start with the Concepts
+If the Geocortex Mobile Designer cannot accomplish what you need, you can manually configure and extend Geocortex Mobile in a number of ways.
```

Comment:
What do you think about including a h2 heading here and explicitly call it `Spectrum of Extensibility`, then change all of the following headings to h3? The reasoning being that it gives some context to the spectrum diagram without having to fully understand the preceding paragraphs.

Reply:
That's quite funny you suggested that - it's what I originallly had and then ended up removing it.

I think a header is definitely useful, but I think my hesistancy was in the wording "Spectrum of Extensibility". I don't explicit mention a "spectrum", or build on that concept any further than in the heading, and that felt weird. Maybe this is the issue, and I should explicitly callout and introduce the "Spectrum of extensibility" in the opening paragraph.

Thoughts? do you think the heading by itself without any other references to that concept is ok?

Reply:
Hmm that's a good point. We seem to explain it implicitly which may be enough? We could add it in and run it by a couple people. Maybe Eric & Ryan?

Reply:
@eric-parsons @rcooney what do you think of having the "Spectrum of Extensibility" header in the overview pages without it being explicitly mentioned.

Here's a link to the preview
https://deploy-preview-51--geocortex-dev-center.netlify.app/docs/web/overview#spectrum-of-extensibility

Reply:
I think "Extending Geocortex Web" would be a better heading name. "Spectrum of Extensibility" only makes sense after you have read all the content. Also we don't really refer to it as a spectrum in the text.

Reply:
I agree with Ryan. I found the original title a bit awkward.

Reply:
Thanks for the feedback @eric-parsons  @rcooney! Changed the titles to "Extending Geocortex Web" and "Extending Geocortex Mobile" respectively.